queries.sql

```diff
@@ -1,3 +1,25 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+select * from animals where RIGHT(nam
+e,3) = 'mon';
+
+select * from animals
```

Comment:
- [ ] You should select only animal names born between 2016 and 2019, instead yo're selecting all the info in the animal table
- [ ] Also kindly make sure the second `date_of_birth` is `2019/31/12` as it is the last day of the year